community/mdadm/build

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+sed -i '/<sys\/types.h>/a #include <sys\/sysmacros.h>' mdadm.h
+sed 's@-Werror@@' -i Makefile
+
+make
+make DESTDIR="$1" install
```

Comment:
One more small thing I failed to notice. The `Makefile` is, of course, dumb. 

the `make` and `make install` lines should be: 
`make BINDIR=/usr/bin UDEVDIR=/usr/lib`
`make DESTDIR="$1" BINDIR=/usr/bin UDEVDIR=/usr/lib install`

Reply:
Good points. I had checked the defaults and didn't think much of them since they symlink to the "correct" destinations. But yeah seems like conflict/alternative directions if they are explicitly in the standard places.

Reply:
They end up in the right place because sbin -> bin -> usr/bin and lib -> usr/lib, but removing packages which *think* they're in /sbin might be... not good :)